common/tools/eslint-plugin-azure-sdk/.eslintrc.json

```diff
@@ -33,6 +33,7 @@
     "no-multi-spaces": "error",
     "no-redeclare": "error",
     "no-useless-escape": "off",
-    "prefer-template": "error"
+    "prefer-template": "error",
+    "sort-imports": 1
```

Comment:
curious why 1 here while others are either "error" or "off"

Reply:
I wasn't able to add "warning" - I only had the options to do 0 1 or 2:

![image](https://user-images.githubusercontent.com/42699471/142696145-37ead478-c1c6-4ab3-bd66-1374bf63d3fe.png)

Reply:
Ah, I see i can change it to "warn", not "warning". Let me fix that.

Reply:
How strange!? I am seeing different suggestion...not sure whether it is related, I have ESLint VS Code extension v2.2.2 installed.  Would it work if you ignore the suggestion and put "warn" there?

![image](https://user-images.githubusercontent.com/7583839/142696673-ac9c641e-cfa8-43fc-9931-f77818c922f4.png)